### PR TITLE
added django 3.0 in tox file

### PR DESCRIPTION
**Pull request by b4isty**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ matrix:
   allow_failures:
     - env: TOXENV=warnings
 
+
```
Comment (auvipy) [inline .travis.yml]: these are not needed.

**Comment by auvipy** [comment]
https://github.com/encode/django-rest-framework/pull/6953/files

File: tox.ini [hunk 2]
```diff
@@ -4,6 +4,7 @@ envlist =
        {py35,py36}-django20,
        {py35,py36,py37}-django21,
        {py35,py36,py37}-django22,
+       {py36, py37}-django30,
```
Comment (Djailla) [inline tox.ini]: No space between versions, keep the other line style

File: tox.ini [hunk 3]
```diff
@@ -24,6 +25,7 @@ deps =
     django20: django==2.0.*
     django21: django==2.1.*
     django22: django==2.2.*
+    django30: django>=3.0a1,<3.1
```
Comment (Djailla) [inline tox.ini]: Try to keep the same style

like `django==3.*`

Comment (rpkilby) [reply tox.ini]: Unfortunately, `django==3.0.*` won't match alpha/beta pre-releases, so it's necessary to use this style until the final version is released.

**Pull request merged**